### DISCUSSION TITLE
Does Bootscore lazy load images?

### DISCUSSION THREAD

**Discussion by bamhost** [bootscore/bootscore#881] [General]
I'm going crazy trying to find the source of what is lazy loading images across the site... the loading="lazy" gets added to every image across the website pages, and all images in woocommerce. I have tried adding the following to the child theme functions to no avail:

add_filter( 'wp_lazy_loading_enabled', '__return_false' );

function disable_lazy_load_featured_images($attr, $attachment = null) {
	$attr['loading'] = 'eager';
	return $attr;
}
add_filter('wp_get_attachment_image_attributes', 'disable_lazy_load_featured_images');

I've went into all plugins that have the setting and disabled it there as well.

I've even tried installing a plugin that is supposed to disable lazy loading completely and that plugin did not work.

So, thought I should check here and also ask if anyone has any suggestions?

Thank you for any help.

**Comment by crftwrk** [comment]
Hi,

all themes and plugins here do nothing with default WP functionality. So, no lazy loading on Bootscore. Quickly checked the filter:

```php
add_filter('wp_lazy_loading_enabled', '__return_false');
```

It works as expected and removes `loading="lazy"` attribute from `<img>` tag.

It must be something in your plugin area or maybe check your server settings?
